src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	public String value;
+    protected boolean isPrivate;
```

Comment:
`isPrivate` doesn't need to be protected as the objects of a subclass must also contain the superclass' private fields. Making this variable protected only makes sense if you wish for a subclass' object to also have access to that field in its superclass.